docs/management_api.yml

```diff
@@ -1,71 +1,153 @@
-swagger: '2.0'
+openapi: 3.0.3
+
 info:
   title: Device Connect
-  version: '1'
   description: |
-    An API for device connect management.
-    Intended for use by the web GUI.  
-
-host: 'hosted.mender.io'
-basePath: '/api/management/v1/deviceconnect'
-schemes:
-  - https
-
-consumes:
-  - application/json
-produces:
-  - application/json
-
-responses:
-  NotFoundError: # 404
-    description: Not Found.
-    schema:
-      $ref: "#/definitions/Error"
-  ConflictError: # 409
-    description: Conflict, record already exists.
-    schema:
-      $ref: "#/definitions/Error"
-  InternalServerError: # 500
-    description: Internal Server Error.
-    schema:
-      $ref: "#/definitions/Error"
-  InvalidRequestError: # 400
-    description: Invalid Request.
-    schema:
-      $ref: "#/definitions/Error"
+    API for managing persistent device connections.
+    Intended for use by the web GUI
+
+  version: "1"
+
+  contact:
+    name: Mender Support
+    email: support@mender.io
+    url: https://mender.io/contact-us
+
+  license:
+    name: Apache 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0.html
+
+servers:
+  - url: https://hosted.mender.io/api/management/v1/deviceconnect
+
+# Global security definitions
+security:
+  - ManagementJWT: []
+
+# Global tags
+tags:
+  - name: ManagementAPI
```

Comment:
Unfortunately, our mender-api-docs site does not recognize the global tags, would it be possible to move under each end-point?